sycl/test-e2e/Matrix/XMX8/element_wise_odd_sizes_bf16.cpp

```diff
@@ -0,0 +1,26 @@
+//==----------- element_wise_all_ops_bf16.cpp  - DPC++ joint_matrix---------==//
```

Comment:
correct name of the file.
Also use similar name to https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/Matrix/joint_matrix_all_sizes_impl.hpp

avoid the word "odd". you can call it element_wise_all_sizes.cpp
no need for bf16 as we should add int8 cases too in the same test.

Reply:
Done